isotovideo

```diff
@@ -519,7 +519,8 @@ sub process_command {
 
     # handle HTTP commands
     if ($rsp->{cmd} eq 'status') {
-        my $result = {tags => $tags, running => $current_test_name};
+        my $result
+          = {tags => $tags, running => $current_test_name, test_execution_paused => $stop_passing_commands_to_backend, pause_test_name => $pause_test_name};
```

Comment:
It would be nice if you would add `$current_test_full_name` as well. This one includes the prefix, eg. 'installation-welcome' instead of just 'welcome'.

Reply:
Please use this route only for initial status, so I guess you should put updates to test_execution_paused too

Reply:
@coolo What do you mean by initial status? Since it's served via websocket it can be used to get current state information. But If I get what you mean correctly:

I could move it to a separate command and subscribe to that one over websockets, and leave status command as is.

Reply:
So my arguments aren't against this change but pro having a message about  test_execution_paused towards the client if the tab is already open.

Reply:
Ah I get it now! Indeed It's gonna be through the [ws proxy](https://github.com/foursixnine/openQA/commit/b24eab7728deb1333661cccc8325cf53a4f2f30d#diff-d607a0943ac197c97f5def97f636753aR309)

Reply:
@coolo Of course. Getting the initial state is the use-case for this.

Further updates are already emitted anyways - at least in my web socket proxy route for the developer mode. This is also already tested. The web socket proxy route which is only for status updates should do the same of course.

Reply:
Btw, both web socket proxy routes should behave the same to simplify the code (on both, server and client). The only difference is that the status-only route doesn't open a developer session and doesn't allow to pass commands to os-autoinst.